### DISCUSSION TITLE
Feature Request[bypass internet connection to install Windows 11]

### DISCUSSION THREAD

**Discussion by crasadure** [ventoy/Ventoy#1661] [General]
Hi @ventoy 
Please firstly can you read this article from this link?
[https://pureinfotech.com/bypass-internet...indows-11/](https://pureinfotech.com/bypass-internet-connection-install-windows-11/)

Can we add a parameter to ventoy.json file for win11 pro and home version for Bypass internet connection 

OOBE\BYPASSNRO

**Comment by steve6375** [comment]
That is not needed, simply type in any invalid email address when asked, it will give an error and then you can create a local user account.
The other way is to use an XML file which will automatically create a local account.

**Comment by Korb** [comment]
@crasadure, related to https://github.com/ventoy/Ventoy/discussions/1861
